player/command.c

```diff
@@ -2364,6 +2364,25 @@ static int mp_property_fullscreen(void *ctx, struct m_property *prop,
     return r;
 }
 
+#ifdef _WIN32
```

Comment:
So its fine to leave option visible and changeable on other platforms even if affects absolutely nothing there?